addons/website_slides/views/website_slides_templates_course.xml

```diff
@@ -243,54 +243,86 @@
     </t>
 </template>
 
+<template id="course_content_training_slide" name="Slide template for a traning channel">
+    <li t-att-index="j" t-attf-slide_id="#{slide.id}" t-attf-category_id="#{category.id if category else None}" class="content-slide slide-draggable d-flex justify-content-between align-items-center p-2">
```

Comment:
j ? Try to remove if possible (or find a real name, like slide_index ).

Reply:
Removed j and also removed a i somewhere.
(There are still some i left to clean I think :policeman: )

Reply:
Nevermind, I'm putting "i" back as fullscreen JS strangely uses that "index" information A LOT.
Tried to change it but honestly the code is brainfucking me so I'll let qmo remove/clean it.